src/isWilsonPrime.js

```diff
@@ -19,7 +19,25 @@
  * @return {boolean}
  */
 function isWilsonPrime(num) {
-  // write code here
+  if ((num ^ 0) !== num || num < 0) {
```

Comment:
побитовые операторы редко используются
Лучше сделать что-то вроде этого
```suggestion
  if (Math.truncate(num) !== num || num < 0) {
```

Reply:
или num % 1 !== 0